### ISSUE TITLE
MAKE ABOUT $80 - $120 DAILY WITH MY BOT.  100% WIN RATE

### ISSUE THREAD

**Issue by POPS-SPEC** [Fruitfella/ROBOT-binary#424]
MAKE $80 - $120 DAILY WITH MY BOT ON DIGIT DIFFERS. THIS BOT IS WITH 100% WIN RATE. INVESTMENT OF $55 AND MAXIMUM WIN = $300. PLEASE INTERESTED INDIVIDUALS SHOULD CONTACT ME VIA MY EMAIL: iamchieffred@gmail.com. com

**Comment by navneetsinh** [comment]
If you've 100% win rate bot so why here is maximum loss = 55$,
100% bot contain no loss, even single,
If your bot get a single loss, it can't be a 100% win rate.

**Comment by POPS-SPEC** [comment]
Sorry you misunderstood my write-up. I really mean '' $55 minimum
investment ''.

**Comment by nobleb2007** [comment]
Hmmm, look interesting. Show a live video of the bot.

**Comment by POPS-SPEC** [comment]
Sorry i don't have a video.  The bot goes for $50 and i can't destroy my
reputation because of $50. Just buy it and enjoy.  Thanks

**Comment by nobleb2007** [comment]
Buy what u didn't not see eh. Is that how to buy something. Thanks

**Comment by reggie66** [comment]
frud.....
if you have a bot that gives you wins, why are you scouting for buyers here?
just be nice and share it...
scammer

**Comment by ceakuk** [comment]
@navneetsinh 
100% win rate mean 0% chance that you will  hit the minimum balance ( loose 55$)
but anyway, i don't believe him

**Comment by navneetsinh** [comment]
@ceakuk yes, in that case only required stake value in balance than just go to play and take profit in $$$$$$$$$$$$$

**Comment by ceakuk** [comment]
@navneetsinh 
he should invest his 80$ to make 180$ profit  on our behalf. then deduct his 55$ and then send us the bot plus the change (125$)

**Comment by lasaidhari** [comment]
Bot news RSI https://github.com/Fruitfella/ROBOT-binary/issues/425#issuecomment-508336769
https://www.youtube.com/watch?v=fCHRaUSOQg8

**Comment by kkptra** [comment]
Hi.. Please share bot for test. @ kkptra@gmail.com . If it is really with100% win rate i will pay you $500... No worries i will also not put my reputation on stack for just $500...

**Comment by ceakuk** [comment]
@lasaidhari 
Virtual account means nothing. maybe that time session was lucky or out of 10  trial that failed  you choose to share the only time that it worked

**Comment by jane327** [comment]
@ceakuk can I communicate with you?I would like to ask you something

**Comment by ZDEGLET** [comment]
ШЛЯПА

**Comment by muroyal** [comment]
> Hi.. Please share bot for test. @ [kkptra@gmail.com](mailto:kkptra@gmail.com) . If it is really with100% win rate i will pay you $500... No worries i will also not put my reputation on stack for just $500...

I will provide you high win rate safe BOT if you pay me 500$ :)

**Comment by muroyal** [comment]
> > Hi.. Please share bot for test. @ [kkptra@gmail.com](mailto:kkptra@gmail.com) . If it is really with100% win rate i will pay you $500... No worries i will also not put my reputation on stack for just $500...
> 
> I will provide you high win rate safe BOT if you pay me 500$ :)

My email account evaroyal84@gmail.com.

**Comment by muroyal** [comment]
https://youtu.be/rSCK9y8I7SA

**Comment by XiaoYuzu** [comment]
funniest scamer ever
